CMakeLists.txt

```diff
@@ -148,11 +108,64 @@ set(CLP_FFI_JS_SRC_ZSTD
     src/submodules/zstd/lib/decompress/zstd_decompress.c
 )
 
-target_sources(
-    ${CLP_FFI_JS_BIN_NAME}
-    PRIVATE
+# List of supported environments.
+set(CLP_FFI_JS_ENVIRONMENTS
+    "node"
+    "worker"
+)
+
+foreach(env ${CLP_FFI_JS_ENVIRONMENTS})
+    set(CLP_FFI_JS_BIN_NAME "ClpFfiJs-${env}")
+    add_executable(${CLP_FFI_JS_BIN_NAME})
+    target_compile_features(${CLP_FFI_JS_BIN_NAME} PRIVATE cxx_std_20)
+    target_compile_options(${CLP_FFI_JS_BIN_NAME} PRIVATE ${CLP_FFI_JS_ENABLE_EXCEPTION_FLAG})
+
+    set(CLP_FFI_JS_LINKER_FLAGS
+        ${CLP_FFI_JS_ENABLE_EXCEPTION_FLAG}
+        ${CLP_FFI_JS_EXTRA_LINKER_FLAGS}
+        -sALLOW_MEMORY_GROWTH
+        -sEXPORT_ES6
+        -sMODULARIZE
+        -sWASM_BIGINT
+        --emit-tsd=${CLP_FFI_JS_BIN_NAME}.d.ts
```

Comment:
Should we also be specifying `sENVIRONMENT=${env}`?

Reply:
Sorry, we should. I think I mistakenly deleted it when I was merging from main.